### PR TITLE
Models setup with initial Seed #40011

### PR DESCRIPTION
**Pull request by shashankbaranwal**
- Fecthing data inside controller and returning response.

### CODE REVIEW

File: src/repositories/user/UserRepository.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    public create(data: any): Promise<IUserModel> {
+        console.log('UserRepository:: create', data);
+        const id = UserRepository.generateObjectId();
+        const model = new userModel({
+            _id: id,
+            ...data,
+        });
+        return model.save();
+    }
+    public count() {
```
Comment (namanpar) [inline src/repositories/user/UserRepository.ts:24]: please include update method as well

File: src/controllers/users/routes.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { Router } from 'express';
+import userController from './controllers';
+import validationHandler from '../../libs/routes/validationHandler';
+import authMiddleWare from '../../libs/routes/authMiddleWare';
+import validation from './validation';
+const userRouter = Router();
+userRouter.route('/')
+    .post(authMiddleWare('getUsers', 'write'), validationHandler(validation.create), userController.create);
+
+export default userRouter;
```
Comment (namanpar) [inline src/controllers/users/routes.ts:10]: define all routes for user controller

**Review by namanpar — CHANGES_REQUESTED** [review]
do the requested changes

File: src/controllers/users/controllers.ts [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                    }
+                ]
+            });
+        }
+        catch (err) {
+            console.log('Inside err', err);
+        }
+    }
+}
+export default userController.getInstance();
```
[the remark this answers is not in the document]
Comment (shashankbaranwal) [reply src/controllers/users/controllers.ts]: Used ES6 and do proper naming function in this files.

File: src/repositories/user/UserRepository.ts (hunk 1 above)
Comment (shashankbaranwal) [reply src/repositories/user/UserRepository.ts:24]: Vinay sir said that only do create and count method

File: src/controllers/users/routes.ts (hunk 2 above)
Comment (shashankbaranwal) [reply src/controllers/users/routes.ts]: Defined all routes for user controller

**Review by namanpar — APPROVED** [review]

**Pull request merged**